src/index.js

```diff
@@ -1175,7 +1189,8 @@ Carousel.defaultProps = {
   width: '100%',
   withoutControls: false,
   wrapAround: false,
-  slideListMargin: 10
+  slideListMargin: 10,
```

Comment:
same comment as above

Reply:
Thanks for the notice, all changes are in the next push